<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: #FFF;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>200</y>
         <width>31</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Dia</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_dia">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>210</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_title">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>110</y>
         <width>421</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 0, 255);
color: #FFF;
border-radius: 10px;</string>
       </property>
       <property name="text">
        <string>Verificar se a data é válida</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_verificar">
       <property name="geometry">
        <rect>
         <x>338</x>
         <y>350</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 0, 255);
color: #FFF;
border-radius: 10px;</string>
       </property>
       <property name="text">
        <string>Verificar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_verificar">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>418</y>
         <width>511</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 0, 255);
color: #FFF;
border-radius: 10px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>315</x>
         <y>279</y>
         <width>31</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ano</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ano">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>290</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>315</x>
         <y>240</y>
         <width>41</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Mês</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_mes">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>250</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
